### PR TITLE
Updates license with info for all title songs

### PR DESCRIPTION
**Pull request by peoplearestrange**
## About The Pull Request
Updates the license.txt with authors and titles for the remaining 3 title.ogg's

## Why It's Good For The Game
People get credit and now their original song titles are easily findable

## Changelog
Not needed

### CODE REVIEW

**Comment by Fikou** [comment]
these arent ambience

**Comment by nemvar** [comment]
Shush, they are in the ambience folder.

**Comment by optimumtact** [comment]
\>assumed cc
yeah and im the queen of londonderry

**Comment by cacogen** [comment]
First rule of copyright is you don't talk about copyright

**Comment by peoplearestrange** [comment]
> >assumed cc
> yeah and im the queen of londonderry

Yeah I guess that was a bit eh... I sent them a forum PM but they haven't been about since 2015.
The sound effects originally made by cuboos were directly made for this code and the song was requested by people to which they supplied. I guess its not completely stated that its CC by the original user, but I'll do my best to obtain clarification.

File: sound/ambience/license.txt [hunk 1]
```diff
@@ -5,3 +5,6 @@ ambidet2.ogg is Night on the Docks, Piano by Kevin Macleod. It has been licensed
 aurora_caelus.ogg is Music for Manatees, by Kevin Macleod. It has been licensed under CC-BY 3.0 license.
 	It has been cropped for use ingame, and also fades out.
 title0.ogg is Endless Space by Solus. It has been licensed under CC-BY 3.0 license. Source file downloaded from https://www.newgrounds.com/audio/listen/74946
+title1.ogg is Flip-Flap created by Jakub "AceMan" Szeląg and taken from http://www.modules.pl/?id=module&mod=453
+title2.ogg is Robocop Theme (gameboy) remixed by Eric Schumacker
+title3.ogg is Tintin On The Moon by Cuboos https://tgstation13.org/phpBB/viewtopic.php?f=10&t=2157 (assumed CC under allowing it to be submitted to the github, see thread)
```
Comment (Qustinnus) [inline sound/ambience/license.txt]: a remix of tintin on the moon*

Comment (peoplearestrange) [reply sound/ambience/license.txt]: Good point! will change

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**